sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/Schema.java

```diff
@@ -949,6 +950,40 @@ public int hashCode() {
             getMetadata()
           });
     }
+
+    @Override
+    public String toString() {
```

Comment:
I am not sure here: without this toString(), what is the default toString output for, say ROW type?

Reply:
This is FieldType, without it, its just a dump of all the fields in a json-ish structure, which makes it hard to determine at a glance what is going on in debug prints, and makes it not a great method to present to end users when saying "this field is badly formatted". This makes the output very similar to the DDL you would have to type to describe a field of this type.

Reply:
I see. In this case, do you think if it is worth also having simple tests for the new toString() code added (just for MAP,ROW,ARRAY)?

Reply:
@dpcollins-google  what do you think of this comment?

Reply:
Will add a test

Reply:
Added